dashboard/test/controllers/api/v1/section_instructors_controller_test.rb

```diff
@@ -12,8 +12,6 @@ class Api::V1::SectionInstructorsControllerTest < ActionController::TestCase
     # These are auto-created for the section creator
     @si1 = @section.instructors.first
```

Comment:
Is `@section`, `@si1`, `@si2`, `@si3` being used anymore? Can we remove those instantiations?

Reply:
`@section` and `@si3` are still referred to in the read-only tests. `@si1` and `@si2` aren't. I'll get rid of those, thanks.